app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/DraftExciseMovementController.scala

```diff
@@ -59,7 +61,11 @@ class DraftExciseMovementController @Inject()(
     }
 
   private def submitMessage(boxId: String)(implicit request: ValidatedXmlRequest[NodeSeq]): Future[Result] = {
-    submissionMessageService.submit(request).flatMap {
+    // TODO when using MessageValidation will have one ern returned after validation
+    // TODO For now, we know the consignor is fine because it passed the validate action above.
+    val submittingErn = request.message.asInstanceOf[IE815Message].consignorId
```

Comment:
can we use MessageValidation now as part of this work? At least for the draft Movement, so we can rid off of this cast.

Reply:
done